scripts/run-ci-e2e-tests.js

```diff
@@ -280,12 +280,16 @@ try {
       exitCode = 1;
       throw Error(exitCode);
     }
-    describe('Test: Flow check');
```

Comment:
Do we need this now that CI is optional?

Reply:
Good point, I can try un-commenting it and see what happens

Reply:
Yep, the CircleCI tests are optional! As such, this PR is no longer needed